ow_lander/scripts/path_planning_commander.py

```diff
@@ -30,166 +29,129 @@ class PathPlanningCommander(object):
   def __init__(self):
     super(PathPlanningCommander, self).__init__()
     moveit_commander.roscpp_initialize(sys.argv)
-    robot = moveit_commander.RobotCommander()
-    scene = moveit_commander.PlanningSceneInterface()
-
-    move_arm = moveit_commander.MoveGroupCommander("arm")
-    # move_limbs = moveit_commander.MoveGroupCommander("limbs")
-    # move_grinder = moveit_commander.MoveGroupCommander("grinder")
-    display_trajectory_publisher = rospy.Publisher('/move_group/display_planned_path',
-                                                   moveit_msgs.msg.DisplayTrajectory,
-                                                   queue_size=20)
-    self.arm_move_group = move_arm
-    # self.move_limbs = move_limbs
-    # self.move_grinder = move_grinder
+
+    self.arm_move_group = moveit_commander.MoveGroupCommander("arm")
+    self.limbs_move_group = moveit_commander.MoveGroupCommander("limbs")
```

Comment:
What does "limbs" refer to? Should be there an antenna group?

Reply:
The limbs group is same as the 'arm' group except it doesn't include the scope.

Good point on the antenna. The antenna handling was not part of this file but maybe it should .. I need to check where and how the antenna service is implemented (I will add a task for this to the epic).